src/main/flight/mixer.c

```diff
@@ -613,11 +619,30 @@ static void calculateThrottleAndCurrentMotorEndpoints(timeUs_t currentTimeUs)
 
         }
 #endif
+
+        // reduce motorRangeMax when battery is full
+        if (vbatSagCompensationFactor > 0.0f) {
+            const float vbatFull = batteryConfig()->vbatmaxcellvoltage - 10;
+            const float vbatLow = batteryConfig()->vbatwarningcellvoltage;
+            const float vbatRangeToCompensate = vbatFull - vbatLow;
+            const float currentCellVoltage = (float)getBatterySagCellVoltage();
+            float batteryGoodness = 0.0f;
```

Comment:
I agree, it is an odd term.  I tossed up many names, but a lot are technical terms with more precise meanings.  

The number indicates where the battery sits in terms of the compensation voltage range, and varies up and down all the time, from 0 - 100.  

It's not the actual battery fill level, which is a name used by quite a few chargers as the residual battery content in mAh.  

It could perhaps be batteryStatus - that's a nice sounding, but bland, term - but I found batteryGoodness quite nice, because it it sort of does just tell you how... good, or bad, the battery is.

Reply:
@mikeller I have updated the names, but decided to leave this one as is, unless you feel strongly about it.  I hope it reflects a bit of a sense of humour, something perhaps reflecting my personal way of thinking about the goodness of what we do.